### PR TITLE
More FPS chart cleanup.

### PR DESCRIPTION
**Pull request by terrylucas**
More cleanup of colors for jank and hover.

### CODE REVIEW

File: packages/devtools/lib/src/timeline/frames_bar_plotly.dart [hunk 1]
```diff
@@ -119,15 +103,11 @@ class FramesBarPlotly {
       name: 'GPU Jank',
       hoverinfo: 'y+name',
       hoverlabel: HoverLabel(
-        font: Font(color: 'white'),
-        bordercolor: 'rgba(255, 0, 0, .7)',
+        font: Font(color: 'black'),
```
Comment (jacob314) [inline packages/devtools/lib/src/timeline/frames_bar_plotly.dart]: make 'black' be a ThemedColor as well

File: packages/devtools/lib/src/timeline/timeline.dart [hunk 2]
```diff
@@ -28,6 +28,14 @@ const Color mainCpuColor = Color(0xFF64B5F6);
 // https://material.io/design/color/the-color-system.html#tools-for-picking-colors.
 const Color mainGpuColor = Color(0xFF4DB6AC);
```
Comment (jacob314) [inline packages/devtools/lib/src/timeline/timeline.dart]: Add a TODO to switch these from Color to ThemedColor so these will work with the Dark theme as well.

**Comment by devoncarew** [comment]
Drive by comment:

```
Checking dartfmt...
Failed dartfmt check: run dartfmt -w bin/ lib/ test/ web/
src/timeline/timeline.dart
```

File: packages/devtools/lib/src/timeline/timeline.dart (hunk 2 above)
Comment (terrylucas) [reply packages/devtools/lib/src/timeline/timeline.dart]: thx. done.

**Pull request merged**